### DISCUSSION TITLE
Should intents handle errors in Speech to Text?

### DISCUSSION THREAD

**Discussion by tannisroot** [OHF-Voice/intents#2400] [_General]
I was thinking of contributing changes that would make intents handle cases where there was a mild error on speech to test processing. 
For example, I've noticed that sometimes my Whisper instance would interpret me saying `Turn off lights` as `Turning off lights`, so it would make sense to change intents to accept "turning" as valid too.
However, is it something that would be actually and accepted as a change? I would like to know before making the changes.

**Comment by tetele** [comment]
In theory, that's what [this](https://github.com/home-assistant/hassil/pull/80) is for and my answer would be that they shouldn't handle mistranscriptions, but i am not aware of any plans to use that feature.

So i'll let Mike answer this one 😅
